### PR TITLE
Add Puck Library guide

### LINKED ISSUE #14: How to use the puck lib (and other guides using nordic pucks)
A guide on how to setup a locatin puck.

If necessary more guides with other pucks

### PR DESCRIPTION
**Pull request by sigvef**
This fixes #14.

image

image

### CODE REVIEW

File: guides/pucklib.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+So you have already created a project in the [online compiler](https://developer.mbed.org/compiler), and you want to add the Puck library.
+Click "Import" in the compiler menu, and search for "Puck".
+The top hit will be the Nordic Puck library.
+Double-click it, and it will automatically be imported to your project.
+
+## Enabling it in the code
+
+The puck library needs to be included in your main.cpp file to work.
+Slap this in the top of your file, and you should be good:
```
Comment (cristeahub) [inline guides/pucklib.md:22]: Is this the right verb? lol ^^

**Comment by aleksanb** [comment]
Good choice of image!

File: guides/pucklib.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```cpp
+Puck* puck = &Puck::getPuck();
+```
+
+Finally, the Puck library acts a cooperative multitasker coordinator, which means that it needs control over the main loop of the program.
+This means that your main loop should look like this:
+
+```cpp
+while(puck->drive()) {
+    /* do stuff here, but nothing that takes too long */
```
Comment (cristeahub) [inline guides/pucklib.md]: what kind of stuff might be done here? should be give an example?

File: guides/pucklib.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+This adds a GATT Service called `16 letter ident.` with a 20-byte read/writable characteristic called `-must be unique-`.  (`stringToUUID` is a helper function included with the Puck library).
+
+## Reacting to GATT Characteristic writes
+
+If you want to react on the mbed when a characteristic is written, you can do that!
+Setting it up is simple:
+
+```cpp
+void onChange(uint8_t* value, uint8_t length) {
```
Comment (stianjensen) [inline guides/pucklib.md]: the value needs to be const, right?

Comment (sigvef) [inline guides/pucklib.md]: That's what you get for typing things from memory :p

**Comment by sigvef** [comment]
ping

**Pull request merged**